data-access/usi-mapping.js

```diff
@@ -0,0 +1,160 @@
+const PIC_1_0 = '50025';
+const PIC_2_0 = '50003';
+const multiUsiUoiMappings = {
```

Comment:
This should probably be pulled from the document and stored as a map in memory at runtime rather than being checked in. There are methods to do this in the other utils.